docs/docs/examples/repl.md

```diff
@@ -34,43 +34,57 @@ Steps
 
         opa run
 
-    Without any data, you can experiment with simple boolean expressions to get the hang of it:
-
-        > a = 1, b = 2, a != b
-        +---+---+
-        | A | B |
-        +---+---+
-        | 1 | 2 |
-        +---+---+
-        > a = 1, b = 2, a = b
+    Without any data, you can experiment with simple expressions to get the hang of it:
+
+        > true
+        true
+        > 3.14
+        3.14
+        > ["hello", "world"]
+        [
+          "hello",
+          "world"
+        ]
+
```

Comment:
This is great!  Start people off with something they know/expect and then move onto the new/cool stuff: queries.